nvme.c

```diff
@@ -819,6 +810,27 @@ static int list(int argc, char **argv, struct command *cmd, struct plugin *plugi
 	struct dirent **devices;
 	struct list_item *list_items;
 	unsigned int i, n, fd, ret;
+	int fmt;
+	const char *desc = "Retrieve basic information for the given device";
+	struct config {
+		char *output_format;
+	};
+
+	struct config cfg = {
+		.output_format = "normal",
+	};
+
+	const struct argconfig_commandline_options opts[] = {
+		{"output-format", 'o', "FMT",
+		 CFG_STRING, &cfg.output_format,
+		 required_argument, "Output Format: normal|json"},
+		{NULL}
+	};
+	argconfig_parse(argc, argv, desc, opts, &cfg, sizeof(cfg));
+	fmt = validate_output_format(cfg.output_format);
+
+	if (fmt != JSON || fmt != NORMAL)
```

Comment:
Should be && instead of || if checking for an error condition; the format can't possibly be both. :)